src/util.js

```diff
@@ -106,3 +107,127 @@ export function format_memory_and_limit(usage, limit) {
         return _("");
     }
 }
+
+// TODO
+export function container_stop(container, timeout) {
+}
+
+export function updateContainers() {
```

Comment:
Please add a comment what precisely this function returns, and an example that shows the structure.